components/eamxx/cime_config/config_component.xml

```diff
@@ -44,10 +44,21 @@
     <group>build_component_scream</group>
     <file>env_run.xml</file>
     <desc>
-      A place to buffer atmchange items for test mods.
+      A place to buffer atmchange items so they can be replayed when the namelist_scream.xml is regenerated. This can also be used for testmods that want to change atmconfig items.
     </desc>
   </entry>
 
+  <entry id="SCREAM_HACK_XML">
```

Comment:
If this is intended to be a feature, then maybe we should find a less scary name. What about `SCREAM_MANUAL_XML` or something like that? The word 'hack' sounds like something that should be fixed at some point... :)

Reply:
Maybe. I think we want to discourage usage unless the person is really an expert, so a somewhat scary name is warranted I think. I'm not even sure test cases that modify case XML values were working properly in the old system. The conditions were it is safe to "hack xml" is when the testcase is basic (SMS) or it's a plain (non-test) case and the user is not xmlchange-ing anything that namelist_defaults.xml refers to.